arch/sim/src/sim/CMakeLists.txt

```diff
@@ -288,4 +288,11 @@ endforeach()
 set(HOSTSRCS ${WINHOSTSRCS})
 
 target_sources(nuttx PRIVATE ${HOSTSRCS})
+
+get_target_property(HOST_COMPILE_OPTIONS nuttx COMPILE_OPTIONS)
+foreach(remove_item IN LISTS SIM_NO_HOST_OPTIONS)
+  list(REMOVE_ITEM HOST_COMPILE_OPTIONS ${remove_item})
+endforeach()
+set_target_properties(nuttx PROPERTIES COMPILE_OPTIONS
+                                       "${HOST_COMPILE_OPTIONS}")
```

Comment:
```suggestion
nuttx_remove_compile_options(${SIM_NO_HOST_OPTIONS})
```

Reply:
nuttx_remove_compile_options not able to solve my problem, can you provide some sample how to make it work?